client/src/components/Visualiser/Controller/GUIMode/GUIOperations/OperationDetails.tsx

```diff
@@ -54,7 +57,7 @@ interface Props {
   executeCommand: (command: string, args: string[]) => string;
 }
 
-const OperationDetails: FC<Props> = ({
+export const OperationDetails: FC<Props> = ({
```

Comment:
Inconsistent use of default exports and named exports for React components.

Reply:
So would you prefer all react components to be default or named exports? I would assume default

Reply:
I think React components should export default in general (it's good practice to have 1 component per file).